### DISCUSSION TITLE
Cuba experience

### DISCUSSION THREAD

**Discussion by rwenzlofsky** [vpnhood/VpnHood#103] [General]
Hi. I am currently living in Cuba, working remotely for a company in Italy. I just wanted to let you know that VPNHood! works great while most OpenVPN providers are blocked by the mobile provider Etecsa ( only a few use obfuscation and are able to bypass the DPI). Wireguard works, but they might block it anytime soon. Thanks for this great solution!
I tried Psiphon, lantern, and many other proxy solutions, but on windows, you always need to run an additional software called Proxifier to tunnel all traffic (and not only the browser).  Thanks!

**Comment by trudyhood** [comment]
Thank you for sharing your experience. We are glad to hear that! VpnHood protocol is almost impossible to detect, and we are still working to make it more transparent. However, you should know that the IP address of our public servers can be blocked too because it is public :) . So you can't count on our public servers. Eventually, you need a private VpnHood server installed by an IT expert. Please read the following page:
https://github.com/vpnhood/VpnHood/wiki/Can-a-network-provider-block-VpnHood

Feel free to share any ideas you have.

**Reply by rwenzlofsky** [reply]
Thanks for your answer. Yes, I was setting up two of my own private Linux servers for that reason. Even if one of my servers gets blocked I still have a backup solution and it is easy to spin up a new one at my hosting provider for only a few dollars. I just wonder if it might be suspicious to the mobile network provider if all traffic during the whole workday (8-10 hours) is going between my Cuban cell phone and the VPN server. You mention something about load balancing when using the VPNHood access server but I did not understand how to set this up? I guess this might be a solution to distribute traffic across several destinations making it a little bit less suspicious?

**Reply by trudyhood** [reply]
It is possible to distribute it between many servers, but we haven't implemented it. Also, acquiring many personal servers is not possible for most users. Load Balancer distributes users between servers, and it does not distribute connections on a single user session.

One way to overcome demographic detection is to exclude some common web servers or IP that are not blocked, maybe Gmail or google.

**Comment by Ghost-Frank-Woods** [comment]
Hi, I'm installing Brena Marvo, but I'm facing this white screen, please tell me what to do??
![image](https://user-images.githubusercontent.com/83070160/180677624-43b31a90-962e-462d-b4cc-4df6fdc44b87.png)

**Reply by bobvhood** [reply]
Hi, Please reinstall it from here and let me does it solve the issue
https://github.com/vpnhood/VpnHood/releases/latest/download/VpnHoodClient-Win.exe

**Reply by Ghost-Frank-Woods** [reply]
Hi, I reinstalled the app, and repaired it, but the problem is still there، This problem occurred after installing Windows. Please help.

**Reply by bobvhood** [reply]
Maybe the problem exists in WebView2. Would you install it from here and let me know the result:
https://go.microsoft.com/fwlink/p/?LinkId=2124703

**Reply by Ghost-Frank-Woods** [reply]
Thank you very much, my problem is solved, thank you
![image](https://user-images.githubusercontent.com/83070160/180977145-6095b9b3-a2ad-4505-a3b1-99f320b395cc.png)

**Reply by Ghost-Frank-Woods** [reply]
What settings should I master for better use and experience?

**Reply by bobvhood** [reply]
There are not many settings. UDP mode is better, so switch to UDP. If it works, just keep using it, but if you face any issues, get back to No UDP.
